src/main/java/edu/wisc/my/messages/controller/MessagesController.java

```diff
@@ -31,6 +35,33 @@
                 response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
             }
         }
+
+    @RequestMapping(value="/datedMessages", method=RequestMethod.GET)    
```

Comment:
Not sure `/datedMessages` is the most expressive way to name this web service. Maybe `/currentMessages`? `/messagesFilteredToCurrent` ?

Which maybe leads to an insight that this should be a nuance on `/messages`, as in taking a `filter` request param where a `toCurrent` value turns on the date filtering?

Should filtering by date be the exception? Maybe `/messages` should filter to current by default, and then there should be an exceptional `/allMessages` or `/messages?filtered=false` that doesn't filter that we might use in e.g. admin views?